tests/integration/bundle/test_tls_bundle.py

```diff
@@ -51,7 +49,7 @@ async def test_tls_encrypted_connection_to_postgres(ops_test: OpsTest):
     logs = await run_command_on_unit(
         ops_test, postgresql_primary_unit, "/charm/bin/pebble logs -n=all"
     )
+    username = f"{PGB}_user_{relation.id}_{ops_test.model.info.name}".replace("-", "_")
     assert (
-        f"connection authorized: user=relation_id_{relation.id} database=waltz"
-        " SSL enabled (protocol=TLSv1.2, cipher=ECDHE-RSA-AES256-GCM-SHA384, bits=256)" in logs
+        f"connection authorized: user={username} database=waltz SSL enabled" in logs
```

Comment:
This was checking the wrong username. Also, I saw that the TLS version was updated to 1.3, so I changed the string to a more generic one in the assertion.